src/system/SystemPacketBuffer.h

```diff
@@ -570,6 +586,16 @@ class DLL_EXPORT PacketBufferHandle
         return buffer;
     }
 
+    void Advance()
+    {
+        PacketBuffer * next = mBuffer->Next_ForNow();
+        if (next != nullptr)
+        {
+            next->AddRef();
+        }
+        Adopt(next);
```

Comment:
Could also be `*this = Hold(next);`?  Or is that larger code?

Reply:
It's identical; I just wrote them in the wrong order. I do intend to take a good look at all the inline methods once the public uses are settled.